scraper/pyproject.toml

```diff
@@ -31,7 +31,7 @@ additional-keywords = ["youtube"]
 [project.optional-dependencies]
 scripts = ["invoke==2.2.0"]
 lint = ["black==24.4.2", "ruff==0.4.10"]
-check = ["pyright==1.1.368"]
+check = ["pyright==1.1.371"]
```

Comment:
If you update this dependency, you should align the version in `.pre-commit-config.yaml`

Reply:
Fixed it in https://github.com/openzim/youtube/pull/265/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R34